cpp/src/arrow/dataset/file_parquet_encryption_test.cc

```diff
@@ -51,9 +53,35 @@ using arrow::internal::checked_pointer_cast;
 namespace arrow {
 namespace dataset {
 
+struct EncryptionTestParam {
+  bool uniform_encryption;  // false is using per-column keys
+  bool concurrently;
+  bool crypto_factory;
+};
+
+std::string PrintParam(const testing::TestParamInfo<EncryptionTestParam>& info) {
```

Comment:
Wouldn't it be more idiomatic to define a `operator<<` for `EncryptionTestParam`?